Resources/Prototypes/Entities/Objects/Tools/tools.yml

```
@@ -307,6 +307,12 @@
         Piercing: 10
     soundHit:
       path: "/Audio/Items/drill_hit.ogg"
+  - type: Tending # Shitmed
+    speed: 0.75
+  - type: Drill # Shitmed
+    speed: 0.75
```

```suggestion
    speed: 1
```

even if its not a medical tool, its a fuckin power drill

---

Iirc in ss13 these numbers also corresponded to the chance of successfull surgery, so this may play a role in the future
On the other hand power drill was a really good substitution to a surgical one anyway so I'm fine with something like 0.9